lib/netsuite/records/base_ref_list.rb

```diff
@@ -2,31 +2,15 @@
 
 module NetSuite
   module Records
-    class BaseRefList
-      include Support::Fields
+    class BaseRefList < Support::Sublist
```

Comment:
I tested it, and it looks like it's working for standard records:

```ruby
NetSuite::Records::BaseRefList.get_select_value(
  recordType: 'serviceSaleItem',
  field: 'taxSchedule'
)
```
![image](https://cloud.githubusercontent.com/assets/7997618/17746244/45560dc4-647d-11e6-9d24-06850ee9f1a9.png)

And for custom records:

```ruby
NetSuite::Records::BaseRefList.get_select_value(
  field: 'custrecord_....',
  customRecordType: {
    '@internalId' => 10,
    '@xsi:type' => 'customRecord'
  }
)

```

![image](https://cloud.githubusercontent.com/assets/7997618/17746157/de7cb1fc-647c-11e6-8d7f-97baa9bb6a3d.png)

Also, I should have noted this when I submitted, but I manually tested examples of almost all of the lists that I tweaked in this way to make sure they were still working. I think there were only one or two that I couldn't test. (I believe they were the BillingSchedule ones.)